utils.js

```diff
@@ -43,4 +43,53 @@ function _in(needle, haystack) {
     return false;
 }
 
-module.exports = {parsePrice, composeRanges, _in};
\ No newline at end of file
+function formatHeader(player, auctionMode) {
+
+    let arrayBuffer = [];
+    let padding = ''
+    //70 for iphone, 124 for desktop
+    for (let i = 0; i < 70 - (player.length + auctionMode.length); i++){
+        arrayBuffer.push(' ');
+    }
+    if (auctionMode === '---') {
+        console.log('---')
+        padding = '     ';
+    }
+    const test = `${player}${arrayBuffer.join('')}${padding}*${auctionMode}*`
+    console.log(test.length)
+    return `${player}${arrayBuffer.join('')}${padding}*${auctionMode}*`;
+}
+
+function formatCapitalCase(sentence) {
+    const words = sentence.split(' ');
+    return words.reduce((acc, val, index) => {
+        if (index == 0) {
+            return acc + val[0].toUpperCase() + val.slice(1).toLowerCase();
+        }
+        return acc + ' ' + val[0].toUpperCase() + val.slice(1).toLowerCase();
+    }, '')
+}
+
+function removeLogTimestamp(log) {
+    return log.slice(log.indexOf(']') + 2);
+}
+
+function formatItemNameForWiki(itemName) {
```

Comment:
Err, please don't do things this way -- we literally have items.json for this purpose. Just look up the key/value to get the wiki URL.

Reply:
Good call, I'll take a look.

Reply:
This reminds me, we should fix the item name match for that item someone mentioned that was wrong -- was it Executioner's Axe? Can fix simply by changing the key in items.json, which will make the lookup match properly and link to the wiki correctly regardless of the difference.

Reply:
Yeah- the user was searching for "Executioners Axe" without an apostrophe.  Should I add these keys then?
```
{
"EXECUTIONER'S AXE":"/Executioner%27s_Axe" //we already have this one
"EXECUTIONERS AXE":"/Executioner%27s_Axe",
"AN EXECUTIONER'S AXE":"/Executioner%27s_Axe" ,
"AN EXECUTIONERS AXE":"/Executioner%27s_Axe",
}
```

Reply:
Perfect, in aliases like you did, yes!